cmd/kubectl-k8ssandra/register/command.go

```diff
@@ -0,0 +1,72 @@
+package register
```

Comment:
question: Was there a reason this cmd package is built entirely differently than the existing ones?

Reply:
There are a few yes. 

Firstly, you've piggybacked off the kubectl flags for the rest of the commands. That doesn't make sense here since flags like "--context", "--namespace" make no sense here given there are two clusters. 

Secondly, it may not actually make sense for this to be a kubectl plugin, particularly for commands like this, so I'm trying to avoid interacting with that part of the model.

Thirdly, this command takes a lot more flags than the others. I don't fully understand how the flags logic in the other commands works, but I used this [doco](https://github.com/spf13/cobra/blob/main/site/content/user_guide.md) from the Cobra lib to figure out how to do it. If you have some documentation for your approach then please share and I'll see if I can conform more closely to the way you've done it. 

I'm open to considering changes here, but they may end up being a refactoring ticket.

Reply:
Not all kubectl commands use --namespace either. Imagine ``kubectl version``, it doesn't care about namespace. Yet it still uses the same structure for accepted parameters. You can give it a namespace, it just doesn't do anything with that information. However, all those command line parameters are registered to ensure you don't code again different behavior to the same flag that's used elsewhere.

Even in k8ssandra-client we don't always use those parameters, for example config building doesn't. But it also doesn't try to override existing parameters. 

The one parameter that would make sense here is ``--kubeconfig`` which is in the defaults and is actually what you have to default to in any case with "--source-kubeconfig``, using the same behavior. I'm not sure I see the value in replacing the name.

Also, you really should use RunE to get better error handling.